bundles/binding/org.openhab.binding.powermax/src/main/java/org/openhab/binding/powermax/internal/connector/PowerMaxTcpConnector.java

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright (c) 2010-2016, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.powermax.internal.connector;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.net.Socket;
+import java.net.SocketAddress;
+import java.net.SocketException;
+import java.net.UnknownHostException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A class for the communication with the Visonic alarm panel through a TCP connection
+ *
+ * @author lolodomo
+ * @since 1.9.0
+ */
+
+public class PowerMaxTcpConnector extends PowerMaxConnector {
+
+    private static final Logger logger = LoggerFactory.getLogger(PowerMaxTcpConnector.class);
+
+    private String ipAddress = "";
+    private int tcpPort;
+    private int connectTimeout;
```

Comment:
these could also be marked final if only set in the constructor, and don't need to be initialized here.